### PR TITLE
Add cmakelist

### PR DESCRIPTION
**Pull request by rajkan01**
- Added `CMakeLists.txt` file to BLE examples
- Updated `main.cpp` file refer headers from `mbed-os-ble-utils` without the prefix of its directory name
- Updated circleci config to invoke BLE examples with CMake

Preceding PR# https://github.com/ARMmbed/mbed-os/pull/13948

@0xc0170 @paul-szczepanek-arm

### CODE REVIEW

**Comment by rajkan01** [comment]
I am looking this circle ci failure with cmake builds

**Review by jamesbeyond — APPROVED** [review]
CircleCI looks good, glad to see Cmake works

File: BLE_GattServer_CharacteristicUpdates/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+)
+
+target_link_libraries(${APP_TARGET}
+    PRIVATE
+        mbed-os
+        mbed-events
+        mbed-ble
+        mbed-ble-cordio
+        mbed-ble-blue_nrg
+        mbed-os-ble-utils
```
Comment (0xc0170) [inline BLE_GattServer_CharacteristicUpdates/CMakeLists.txt:41]: While we have this in here, I assue this example should have only `mbed-ble` + `mbed-ble-utils`. the rest should be added via dependencies, correct?

`mbed-ble-utils` rename needed, we do not need `mbed-os` prefix?

**Review by 0xc0170 — APPROVED** [review]

**Comment by rajkan01** [comment]
Preceding PR# ARMmbed/mbed-os#13948

File: BLE_SecurityAndPrivacy/source/main.cpp [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 #include <events/mbed_events.h>
 #include "ble/BLE.h"
-#include "mbed-os-ble-utils/pretty_printer.h"
+#include "pretty_printer.h"
```
Comment (0xc0170) [inline BLE_SecurityAndPrivacy/source/main.cpp:19]: why is this needed ? Or should it be as it was and we add additional include path to CMake where utils.lib is provided.

Rethorical questions if our external libraries should use same dir structure (providing directory in the root with the same name as library that would allow to use `name_library/include.h`)

**Comment by 0xc0170** [comment]
> Preceding PR# ARMmbed/mbed-os#13948

merged. So this now waits for @paul-szczepanek-arm review
